### ISSUE TITLE
Brook Wingman PS2

### ISSUE THREAD

**Issue by Invictaz** [isage/vixen#12]
Tested it with Dualshock 4 v2 connected

[vixen.txt](https://github.com/isage/vixen/files/12491541/vixen.txt)

**Comment by isage** [comment]
um. how do you connect it to pstv?
also, report seems to be missing analogs and L3/R3 (left analog works as dpad). there's probably should be an option to switch to analog mode.

**Comment by Invictaz** [comment]
There is a micro USB port on the back that can connect via USB-A cable to the PSTV. I did input the L3 and R3. The analog mode is disabled in usb mode. It apparently only works if you connect it via the Playstation male port (via an adapter). I tried to convince Brook to enable it via firmware but they will not.
Setting the analog mode in USB should be possible by pressing the home button, but it does not do that for some reason. 

Another thing:

If the PSTV emulates a PC then the Brook adapter switches to a different mode and it will accept analogs and L3/R3. If it does not detect a PC it switches to PS Classic mode and then it does not have analogs and L3/R3. So the only solution is to improve the kernel driver of Vixen to mimic a PC. 

Or hack the firmware of the Brook adapter...

Maybe you can work with what we have now, otherwise we have nothing.

Another workaround is to trick the Brook adapter that it is connected to a PS2 via a USB to PS2 adapter. Then all buttons work. But it is a stupid workaround because of a firmware limitation.

The workaround is to trick the Brook adapter via USB to ps2 adapter so that it thinks it is connected to a PS2. That works on the PS3 but might add lag.

**Comment by isage** [comment]
Well, that explains why it has same reports as psclassic. I wonder, how it detects pc/non-pc, since there's nothing identifying on host side. 
Does it use their driver on PC? Maybe they send some init command to it

**Comment by isage** [comment]
> If you are a PC gamer, press and hold OPTION + D pad-Down button for 2
seconds after inserting your Wingman PS2 to the PC USB port. The white
signal flashes once and it enters PC mode (X-input)

**Comment by Invictaz** [comment]
Yes I tried that. But it doesn't work. Brook even issued a firmware update which improved nothing. They said supporting analog over USB is not the scope of the product. Maybe because it hampers sales on the XE2 they also want to sell?
